proprietary/tokens/src/common/amsterdam/hyphenation.tokens.json

```diff
@@ -0,0 +1,10 @@
+{
+  "amsterdam": {
+    "hyphenation": {
+      "hyphenate-limit-chars": { "value": "12 8 4" },
+      "hyphens": { "value": "auto" },
+      "overflow-wrap": { "value": "break-word" },
```

Comment:
Should we use `overflow-wrap` or `word-break: break-word`? I think I’d prefer the latter as it fills up the first line as much as possible. See the second example [here on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_text/Wrapping_breaking_text#breaking_long_words). @dlnr Did you investigate this?

Reply:
`word-break: break-word` is deprecated, we shouldn't use that. The example uses `word-break: break-all`. `word-break: break-all` beats `hyphens: auto` though, so we can't really use that either 😅

Reply:
I've tried "auto-phrase" but it did not seem to do anything.
> Has the same effect as word-break: normal except that language-specific analysis is performed to improve word breaks by not placing them in the middle of natural phrases.